chromadb/db/mixins/sysdb.py

```diff
@@ -428,9 +432,15 @@ def get_collections(
                 id = self.uuid_from_db(str(collection_id))
                 rows = list(collection_rows)
                 name = str(rows[0][1])
-                configuration = CollectionConfiguration.from_json_str(rows[0][2])
-                dimension = int(rows[0][3]) if rows[0][3] else None
                 metadata = self._metadata_from_rows(rows)
+                dimension = int(rows[0][3]) if rows[0][3] else None
+                if rows[0][2] is not None:
```

Comment:
nit - comment this migration here with the version / date etc

Reply:
Added comment, let's briefly discuss what version this would be when released (maybe date is OK).